storage/myisam/sort.c

```diff
@@ -1,4 +1,5 @@
 /* Copyright (c) 2000, 2016, Oracle and/or its affiliates. All rights reserved.
+   Copyright (c) 2018, Percona and/or its affiliates. All rights reserved.
```

Comment:
Should (C) be for Percona, or both MariaDB and Percona here?

Reply:
Fixed.